subsys/bluetooth/common/Kconfig

```diff
@@ -79,14 +79,20 @@ config BT_BUF_ACL_RX_SIZE
 
 config BT_BUF_ACL_RX_COUNT_EXTRA
 	int "Number of extra incoming ACL data buffers"
-	default 0
+	range 1 65535 if BT_CONN
 	range 0 65535
+	default BT_MAX_CONN if BT_CONN
+	default 0
```

Comment:
Not sure I follow this: 
```
default BT_MAX_CONN if BT_CONN
```

If `BT_CONN=n`, then why we do have any ACL buffers?

Reply:
Ah, yeah that makes sense. Could you add that as a comment for the Kconfig option?